src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperService.java

```diff
@@ -69,6 +69,8 @@
 
     void reviveFloodlight(String region);
 
+    String getContainerIp(String containerName);
```

Comment:
why on earth will you put this local-only code into lockkeeper and request container name over http? Do it directly in test classes
`def dockerClient = DefaultDockerClient.fromEnv().build()`

Reply:
in this case we will not be able to run grpc tests from remote machine

Reply:
Maybe it's cleaner to do sth like `DefaultDockerClient.fromEnv().uri(whereTheDockerIs).build()` in this case? I mean it fixes the issue with container ip for grpc, but we have other tests that manipulate with docker containers and they remain unfixed